### PR TITLE
Set max-age option to be part of static resource GET response

### PR DESCRIPTION
**Pull request by anttiylitokola**

### CODE REVIEW

**Comment by anttiylitokola** [comment]
@yogpan01, @TeroJaasko

**Comment by ciarmcom** [comment]
Verification build successful.

**Comment by anttiylitokola** [comment]
rebuild

**Comment by yogpan01** [comment]
+1

File: source/libNsdl/src/sn_grs.c [hunk 1]
```diff
@@ -601,8 +603,18 @@ extern int8_t sn_grs_process_coap(struct nsdl_s *nsdl_handle, sn_coap_hdr_s *coa
...
+                response_message_hdr_ptr->options_list_ptr = handle->sn_grs_alloc(sizeof(sn_coap_options_list_s));
+                if (response_message_hdr_ptr->options_list_ptr) {
+                    response_message_hdr_ptr->options_list_ptr->max_age_ptr = handle->sn_grs_alloc(1);
+                    if (response_message_hdr_ptr->options_list_ptr->max_age_ptr) {
+                        response_message_hdr_ptr->options_list_ptr->max_age_ptr[0] = 0;
+                        response_message_hdr_ptr->options_list_ptr->max_age_len = 1;
+                    }
+                }
+            }
         }
```
Comment (TeroJaasko) [inline source/libNsdl/src/sn_grs.c]: Most paths in this function seem to abort on out of memory case and return SN_NSDL_FAILURE. If this case is special and does not need that, please add some comment about it.

**Comment by TeroJaasko** [comment]
+1

**Pull request merged**